service/controller/chart/resource/release/create_test.go

```diff
@@ -75,6 +75,8 @@ func Test_Resource_Release_newCreate(t *testing.T) {
 			HelmClient: helmclienttest.New(helmclienttest.Config{}),
 			K8sClient:  k8sfake.NewSimpleClientset(),
 			Logger:     microloggertest.New(),
+
+			TillerNamespace: "giantswarm",
```

Comment:
Not for now but I'm wondering if we should drop the unit tests apart from the desired state.

I think we already did that in app-operator. I'd rather extend our existing integration tests or add new ones. WDYT?

Reply:
Good call, I would delete them in next PR.